template/helm.go

```
@@ -71,18 +69,9 @@ func (t TemplateHelmChartTask) Run() error {
 			}
 
 			buildInfo := BuildInfo{SHA: t.sha}
+			templatedContents, err := SafeTemplate(buildInfo, contents)
```

This error is not handled.